### DISCUSSION TITLE
where can i find dxvk.config in windows 10?

### DISCUSSION THREAD

**Discussion by davipinhati** [Sporif/dxvk-async#58] [Q&A]
im testing dxvk-async in some games but dont know for sure if the async function is on cuz i dont have a dxvk.config next my .exe, where i can find it? i need create it in a notepad and use dxvk.config in txt?

**Comment by begin-theadventure** [comment]
Create `dxvk.conf` in the same directory and add config options to it.

**Reply by davipinhati** [reply]
> Create `dxvk.conf` in the same directory and add config options to it.

a txt file?

**Reply by begin-theadventure** [reply]
Yes, a text file simply named `dxvk.conf`.
